nemo/core/classes/common.py

```diff
@@ -242,6 +242,12 @@ class Serialization(ABC):
     @classmethod
     def from_config_dict(cls, config: DictConfig, *args: Any, **kwargs: Any):
```

Comment:
@okuchaiev This change is necessary for variable interpolation to work properly (`${model.model_defaults.*}` injection). So only the actual train configs will allow such interpolation. All files serialized as .nemo will lose such interpolation support (as the values will be in-place replaced by actual values from above interpolation).

Reply:
any ideas why "*args and **kwargs" crept in? Can we remove them from method signature and the hyndra call below?

Reply:
Hmm this is new. Let me check the history.

Reply:
The TTS collection pr introduced it. In any case, it's probably safe to revert.

Reply:
Why? It enables overrides that I need for TTS models.

Reply:
The overrides are unecessary since TTS can perform its augmentation of the cfg inplace by writing into the struct. The current way I do it, you get full support without breaking any code (and without adding **kwargs) support to from_config_dict.

Primarily, graphs have a rigid condition - no *args and **kwargs. Therefore we cannot support their use in most cases.